src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -209,6 +212,17 @@ private CommandLineSourceRepositoryProvider GetSourceRepositoryProvider()
             return new CommandLineSourceRepositoryProvider(SourceProvider);
         }
 
+        private DependencyBehavior GetSettingsDependencyBehavior(Configuration.ISettings settings)
+        {
+            DependencyBehavior ret;
+
+            string depSetting = SettingsUtility.GetConfigValue(settings, "dependencyVersion");
+            if (!Enum.TryParse<DependencyBehavior>(depSetting, out ret))
```

Comment:
This should use ignore case when parsing the enum
https://msdn.microsoft.com/en-us/library/dd991317(v=vs.110).aspx